Automattic/jetpack
# [WordAds]: Move from raw HTML tags to Jetpack components

This PR moves raw HTML tags to the use of Jetpack components.
Addressed from 80-gh-Automattic/wordads-issues

#### Changes proposed in this Pull Request:

* The goal it's to use those components in future WordAds pages/components, like Earnings and Payments.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Enable WordAds shared UI adding `define( 'ENABLE_WORDADS_SHARED_UI', true );` on `tools/docker/wordpress/wp-config.php` file.
* Go to Jetpack  > WordAds, check if it shows as the image below.
* Try to toggle the button, enabling and disabling WordAds.

![image](https://user-images.githubusercontent.com/692065/165601046-449bc02f-2bfc-474d-91f5-617e2a35eeee.png)
